config/initializers/event_bus_mailers.rb

```diff
@@ -0,0 +1,9 @@
+
+EventBus.listen('new_comment_event') do |event|
```

Comment:
I've changed this slightly to make the naming conventions a little more predictable.

For service calls, "#{model_class}_#{action_name}" ("comment_create")
For event calls, "#{event_kind}_event" ("new_comment_event")

I'm going to start emitting broadcasts for most service calls, as well as for event creation. This'll let us hook into most anything that's happening in the app pretty easily.